infra/ansible/playbooks/batcher.yaml

```diff
@@ -0,0 +1,141 @@
+- name: Run setup playbook
+  ansible.builtin.import_playbook: setup.yaml
+  vars:
+    host: batcher
+
+- name: Run webserver playbook
+  ansible.builtin.import_playbook: webserver.yaml
+  vars:
+    host: batcher
+
+- name: Run go playbook
+  ansible.builtin.import_playbook: go.yaml
+  vars:
+    host: batcher
+
+- name: Run rust playbook
+  ansible.builtin.import_playbook: rust.yaml
+  vars:
+    host: batcher
+
+- name: Run eigenlayer-cli playbook
+  ansible.builtin.import_playbook: eigenlayer-cli.yaml
+  vars:
+    host: batcher
+
+- hosts: batcher
+
+  tasks:
+    # Install required packages
+    - name: Update apt and install required system packages
+      become: true
+      apt:
+        pkg:
+          - pkg-config
+          - libssl-dev
+        state: latest
+        update_cache: true
+      vars:
+        ansible_ssh_user: "{{ admin_user }}"
+
+    # Create directories for each service
+    - name: Create directories for each service if do not exist
+      ansible.builtin.file:
+        path: /home/{{ ansible_user }}/repos/batcher
+        state: directory
+        mode: '0755'
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+
+    # Clone Aligned repository for the service
+    - name: Clone Aligned repository
+      ansible.builtin.git:
+        repo: https://github.com/yetanotherco/aligned_layer.git
+        dest: /home/{{ ansible_user }}/repos/batcher/aligned_layer
+        version: batcher_ansible
+        recursive: false
+
+    # Build the batcher
+    - name: Install deps
```

Comment:
Might be missing a `become: {{ ansible_user }}` here?

Reply:
It is not needed I think